packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -284,6 +284,40 @@ void main() {
     skip: kIsWeb, // [intended] the web handles this on its own.
   );
 
+  testWidgets('can get text selection color initially on desktop', (WidgetTester tester) async {
+    final TextEditingController controller = TextEditingController(
+      text: 'blah1 blah2',
```

Comment:
I think it's easier to set the selection here to be [0, 11) or something then focus the text field by calling `focusNode.requestFocus`.

Reply:
Thanks for the suggestion, this helped create a useful golden test as well